app/controllers/events_controller.rb

```diff
@@ -0,0 +1,37 @@
+class EventsController < ApplicationController
+  before_action :logged_in_user, only: [:new, :create]
+  def new
+    @event = Event.new
+  end
+
+  def create
+    unless current_user.try(:superadmin?)
```

Comment:
yeah, it's annoying - that &&/and thing is one where we have to ignore codeclimate - I will open a chore to adjust our codeclimate.yml to not flag that - apologies for the inconvenience: https://www.pivotaltracker.com/story/show/142133741